Code/cmake/Modules/RDKitUtils.cmake

```diff
@@ -31,46 +36,46 @@ macro(rdkit_library)
     ${ARGN})
   CAR(RDKLIB_NAME ${RDKLIB_DEFAULT_ARGS})
   CDR(RDKLIB_SOURCES ${RDKLIB_DEFAULT_ARGS})
-  if(MSVC)
+  if(RDK_INSTALL_STATIC_LIBS AND NOT RDK_INSTALL_DYNAMIC_LIBS)
```

Comment:
Yeah, that was the idea, being consistent with Linux and Windows. If you build both static and dynamic libraries you would have the `.lib` dynamic import libraries and the `_static.lib` static libraries, just as on macOS and Linux you have the `.so`/`.dylib` dynamic libraries and the `_static.a` static libraries.
If you don't particularly fancy that, we can leave `.lib` files all with the same name, no matter if static or dynamic, and just install them to different folders (e.g., make a `dll` folder under Windows).
Please let me know which solution you'd prefer and I'll modify the PR accordingly.

Reply:
Any thoughts as to which is more "Windowsy"?

Reply:
IMHO it would be better to stick to the `_static` suffix paradigm for all platforms. This avoids the ambiguity of having the same filename/extension on files that actually have different content and purposes.
There is no official Windows convention regarding static/dynamic library naming, as discussed here:
https://stackoverflow.com/questions/2140129/what-is-proper-naming-convention-for-msvc-dlls-static-libraries-and-import-libr